tools/Customization/DevHome.Customization/Views/DevDriveInsightsView.xaml

```diff
@@ -0,0 +1,148 @@
+<UserControl
+    x:Class="DevHome.Customization.Views.DevDriveInsightsView"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:devViewModels="using:DevHome.Customization.ViewModels.Environments"
+    xmlns:devDriveModels="using:DevHome.Customization.Models.Environments"
+    xmlns:converters="using:CommunityToolkit.WinUI.Converters"
+    xmlns:ctControls="using:CommunityToolkit.WinUI.Controls"
+    mc:Ignorable="d"
+    Loaded="UserControl_Loaded">

```

Comment:
You should bind this to a RelayCommand directly on the ViewModel, so that you don't need the intermediate event handler on the View. It will also prevent any concurrency issues if a user were to switch between pages really quickly.